packages/devtools_extensions/integration_test/test/simulated_environment_test.dart

```diff
@@ -192,6 +194,60 @@ Future<void> _testShowBannerMessage(
   await _clearLogs(tester, simController);
 }
 
+Future<void> _testCollapseEnvironmentPanel(
+  WidgetTester tester,
+  SimulatedDevToolsController simController,
+) async {
+  final split = tester.widget<Split>(find.byType(Split));
+
+  final divider = find.byKey(split.dividerKey(0));
+  final environmentPanel = split.children[1];
+
+  final RenderBox environmentPanelRenderBox =
+      tester.renderObject(find.byWidget(environmentPanel));
+  final double environmentPanelRenderBoxWidth =
+      environmentPanelRenderBox.size.width;
+
```

Comment:
at this point, can you add an `expect` to check that the width is what we'd expect? The value of `_VmServiceConnection._totalControlsWidth + 2 * defaultSpacing`?

Reply:
Thanks for the review. 

I added this but it's failing with this message:

```
Expected: <439>
Actual: <435.7075>
```

Looking into why.

Reply:
The width from the `RenderBox` wasn't the expected width but close (not sure why).  
So in https://github.com/flutter/devtools/pull/7062/commits/9c6a468e4b84784d3208ec20818bd071c14d5811, I updated the tests to get the width from the `SizedBox` wrapping the expansion panel.